### DISCUSSION TITLE
Documentation for V5 ?

### DISCUSSION THREAD

**Discussion by sigmarsson** [angularsen/UnitsNet#1185] [Q&A]
Thanks for your work. 

Will you provide a short guideline how to adapt existing v4 code to v5 ?

<img width="224" alt="image" src="https://user-images.githubusercontent.com/43340959/213823354-0e98b831-de7d-46e7-a7ae-152e39d20737.png">

**Comment by angularsen** [comment]
There is a wiki linked from the top of the readme.
See if it helps, or if you are still wondering how to proceed.

Also, while on recent 4.x versions, you will get obsolete warnings that help describe what you should use instead.
https://github.com/angularsen/UnitsNet/wiki/Upgrading-from-4.x-to-5.x
